### DISCUSSION TITLE
Ring Bridge v11.0.6 HOOBS Not Found

### DISCUSSION THREAD

**Discussion by alvinpetitt** [dgreif/ring#982] [Q&A]
I've been having issues with the Ring plugin for HOOBS (v4.2.5, Node: 16.15.1, Homebridge: 1.4.1) since after 11.0.2. I'm trying to get v11.0.6 to see if the issues are fixed. But the update only sees v11.0.5 as the latest. However, it seems that v11.0.6 was released 6 days ago, as of this writing. Is there a way for me to force HOOBS to see 11.0.6?

As far as the issues, it (Homekit) simply stops being able to control my Ring devices, alarm stuck on "arming", lighting won't turn on/off, no video from doorbell, etc. If I restore a HOOBS backup that had 11.0.2 installed, all works as expected. And, of course, when I tried to restore that backup again, it cycled out as I'm only keeping 3 backups and this one was the 4th.

**Comment by dgreif** [comment]
I suggest you reach out to HOOBS to figure out why the latest version isn't visible in their UI.  Hopefully everything else gets sorted out once you are updated.
